system_test/aest_hard_fork_SUITE.erl

```diff
@@ -729,6 +734,10 @@ get_block_by_height(NodeName, Height, Cfg) ->
     {ok, 200, B} = aest_nodes:http_get(NodeName, int_http, [v2, block, height, Height], #{}, Cfg),
     B.
 
+get_block_by_hash(NodeName, Hash, Cfg) ->
+    {ok, 200, B} = aest_nodes:http_get(NodeName, int_http, [v2, block, hash, Hash], #{}, Cfg),
```

Comment:
I see now external https://github.com/aeternity/epoch/compare/v0.11.0...5613ea905d5069e46ccc8b707133d895294c0109#diff-89b8ec7c2f28f3e3c7d21ca9bf070bb7R1325 though good you kept in line with v0.11.0. Probably we shall change all Docker images in this test suite from `:local` to `:v0.11.0` (though not strictly needed for this PR) until https://www.pivotaltracker.com/story/show/156846658